pkg/base1/cockpit.js

```diff
@@ -1280,6 +1280,10 @@ function basic_scope(cockpit, jquery) {
         return deferred.promise;
     };
 
+    cockpit.resolve = function resolve(result) {
```

Comment:
@mvollmer Yeah actually I was wondering what is the purpose of `resolve` and `reject` handling arbitrary `...arguments`. 

My second question then is if this is necessary also for this simple helpers - given that the current `cockpit.reject` helper handles only single argument?